src/builtin.js

```diff
@@ -987,16 +987,8 @@ Sk.builtin.raw_input = function (prompt) {
 Sk.builtin.input = Sk.builtin.raw_input;
 
 Sk.builtin.jseval = function jseval (evalcode) {
```

Comment:
we should kill this or at least make it a compile options this is downright dangerous